File: app/src/main/java/com/hfad/avc/ui/main_avtivity/IMainViewModel.java
package com.hfad.avc.ui.main_avtivity;

import moxy.MvpView;
import moxy.viewstate.strategy.AddToEndSingleStrategy;
import moxy.viewstate.strategy.StateStrategyType;

@StateStrategyType(AddToEndSingleStrategy.class)
public interface IMainViewModel extends MvpView {

}

File: app/src/main/java/com/lastaurus/automatic_congratulations/ui/main_avtivity/IMainViewModel.kt
package com.lastaurus.automatic_congratulations.ui.main_avtivity

import moxy.MvpView
import moxy.viewstate.strategy.AddToEndSingleStrategy
import moxy.viewstate.strategy.StateStrategyType

@StateStrategyType(AddToEndSingleStrategy::class)
interface IMainViewModel : MvpView